docs/editors/new-editor.md

```diff
@@ -79,6 +79,13 @@ example, in vim-lsc the `window/logMessage` notification is always displayed in
 the UI so `-Dmetals.status-bar=log-message` can be configured to direct
 higher-priority messages to the logs.
 
+### `-Dmetals.verbose`
```

Comment:
A minor thing: this flag has a very generic name, but it only applies to bloop.
Maybe it could be used to control also Metals own logging level?

Reply:
We can refine this if users need more refined control over logging specific parts. For now I think it's best to keep it as simple as possible, "You have a problem? Try `-Dmetals.verbose=on`".